src/platform/site-wide/user-nav/components/HelpMenu.jsx

```diff
@@ -16,6 +16,13 @@ function HelpMenu({ clickHandler, cssClass, isOpen }) {
     : FACILITY_LOCATOR_URL;
   const icon = <IconHelp color="#fff" role="presentation" />;
 
+  // eslint-disable-next-line no-console
+  console.log(
+    isVATeamSiteSubdomain(),
+    facilityLocatorUrl,
+    FACILITY_LOCATOR_URL,
+  );
```

Comment:
I think the issue (if I understand correctly) is actually on line 8. We have a hardcoded list of VA subdomains here - `src/platform/utilities/environment/va-subdomain.js`. I think that function should be changed to return true if the domain isn't `(www|staging|dev).va.gov`.  think it's returning false for subdomains not listed, leading to the Facility Locator appearing relative to the subdomain.

Reply:
I added all the new subdomains already but it didn't work in this [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/12558)

Reply:
Ah okay. I think it does work actually, but didn't make it into the release?

Reply:
It did, so I'm not sure. I'm going to add your suggestion and maybe add an unit test for it

Reply:
Oh, for some reason the commits aren't listed under yesterday's release. Weird

Reply:
I copy/pasted `isVATeamSiteSubdomain` into the console on that domain and it returned true, so I don't think that's the problem

Reply:
I actually don't think it was deployed yesterday. It looks like it failed - https://dsva.slack.com/archives/C0MQ281DJ/p1588875279132900. A content deployment happened later but that wouldn't have published this.

Reply:
I did it too and it worked...

Reply:
I don't think yesterday's release went out. I think the header on patientsafety.va.gov will work after today's deployment.